maven build failure

Using the `develop` branch on a clean clone of this repo:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 12.160 s
[INFO] Finished at: 2017-01-13T16:04:08-08:00
[INFO] Final Memory: 11M/133M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ld4p-tracer-bullets: Could not resolve dependencies for project stanford:ld4p-tracer-bullets:jar:1.0-SNAPSHOT: Could not find artifact com.oracle:ojdbc14:jar:10.2.0.2.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

- Could not find artifact com.oracle:ojdbc14:jar:10.2.0.2.0 in central maven repo

--- a/README.md
+++ b/README.md
@@ -46,6 +46,69 @@ All files are moved to the Archive/.. folder for future reference.
 /s/SUL/Bin/LD4P/TB1/bin/do_pipeline.ksh
 ```
 
+##Dependencies
+
+- Java 8
+- Maven 3
+
+The Oracle JDBC maven artifacts require a license, follow the instructions at:
+- http://docs.oracle.com/middleware/1213/core/MAVEN/config_maven_repo.htm
+- https://blogs.oracle.com/dev2dev/entry/oracle_maven_repository_instructions_for
+
+Once the Oracle sign-up/sign-in and licence agreement is accepted, add the sign-in
+credentials to maven settings.  Follow maven instructions to encrypt the passwords, see
+- https://maven.apache.org/guides/mini/guide-encryption.html
+  - encrypt a master password:
+
+        $ mvn --encrypt-master-password
+        Master password:
+        {L+bX9REL8CAH/EkcFM4NPLUxjaEZ6nQ79feSk+xDxhE=}
+
+  - add your master password to `~/.m2/settings-security.xml` in a block like:
+
+        <settingsSecurity>
+            <master>{L+bX9REL8CAH/EkcFM4NPLUxjaEZ6nQ79feSk+xDxhE=}</master>
+        </settingsSecurity>
+
+  - encrypt server password:
+
+        $ mvn --encrypt-password
+        Password:
+        {JhJfPXeAJm0HU9VwsWngQS5qGreK29EQ3fdm/7Q7A7c=}
+
+  - add your encrypted password to `~/.m2/settings.xml` using this template:
+
+        <servers>
+          <server>
+            <id>maven.oracle.com</id>
+            <username>your_oracle_username</username>
+            <password>{JhJfPXeAJm0HU9VwsWngQS5qGreK29EQ3fdm/7Q7A7c=}</password>
+
+            <configuration>
+              <basicAuthScope>
+                <host>ANY</host>
+                <port>ANY</port>
+                <realm>OAM 11g</realm>
+              </basicAuthScope>
+              <httpConfiguration>
+                <all>
+                  <params>
+                    <property>
+                      <name>http.protocol.allow-circular-redirects</name>
+                      <value>%b,true</value>
+                    </property>
+                  </params>
+                </all>
+              </httpConfiguration>
+            </configuration>
+          </server>
+        </servers>
+
+- For additional information about maven settings, see
+    - https://maven.apache.org/settings.html
+    - https://books.sonatype.com/nexus-book/reference/_adding_credentials_to_your_maven_settings.html
+
+
 ##Compiling and Executing the pre- and post-marc processing Java packages (and submitting code coverage data)
 
 Import the project into a Java IDE and use the provided iDE build tools. Don't forget to set the Maven property 

--- a/pom.xml
+++ b/pom.xml
@@ -102,6 +102,28 @@
             </plugin>
         </plugins>
     </reporting>
+
+    <!--http://docs.oracle.com/middleware/1213/core/MAVEN/config_maven_repo.htm-->
+    <repositories>
+        <repository>
+            <id>maven.oracle.com</id>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <url>https://maven.oracle.com</url>
+            <layout>default</layout>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>maven.oracle.com</id>
+            <url>https://maven.oracle.com</url>
+        </pluginRepository>
+    </pluginRepositories>
+
     <dependencies>
         <!--<dependency>-->
             <!--<groupId>org.apache.logging.log4j</groupId>-->
@@ -127,9 +149,9 @@
         </dependency>
         <!-- https://mvnrepository.com/artifact/com.oracle/ojdbc14 -->
         <dependency>
-            <groupId>com.oracle</groupId>
-            <artifactId>ojdbc14</artifactId>
-            <version>10.2.0.2.0</version>
+            <groupId>com.oracle.jdbc</groupId>
+            <artifactId>ojdbc7</artifactId>
+            <version>12.1.0.2</version>
         </dependency>
         <!-- https://mvnrepository.com/artifact/net.sf.saxon/Saxon-HE -->
         <dependency>

